traits/src/router.rs

```diff
@@ -60,12 +60,19 @@ pub trait RouteProvider<AssetId> {
 	}
 }
 
+#[derive(Encode, Decode, Clone, Copy, Debug, Eq, PartialEq, TypeInfo, MaxEncodedLen)]
+pub enum TradeOperation {
+	Sell,
+	Buy,
```

Comment:
```suggestion
	ExactIn,
        ExactOut,
```

Reply:
What is the reason behind the suggestion? Sell/Buy sounds like better operation names.

ExactIn and ExactOut are always known implicit knowledge of Sell and Buys repsectively.